Index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="normalize.css">
+  <link rel="stylesheet" href="stars.css">
+  <title>Stars</title>
+</head> 
+<body>
+    <div class="container">
+      <div class="stars stars--0">
+          <label class="checked">
+            <input type="checkbox" class="stars__star">
+            <span class="check__box"></span>
+          </label>
+          <label  class="checked">
```

Comment:
```suggestion
          <label class="checked">
```
Одного пробела хватит